### PR TITLE
[enterprise-4.14] OSDOCS-5043: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-5369

Link to docs preview: https://65610--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-networking-http-headers

QE review:
- [x] QE has approved this change.

Additional information: xref will not work right now

### CODE REVIEW

**Comment by jldohmann** [comment]
@ShudiLi could you PTAL for QE ack? This PR adds a release note entry for the new feature allowing users to set or delete certain HTTP request and response headers. Thank you!

**Comment by ShudiLi** [comment]
/lgtm
thanks

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -326,6 +326,22 @@ Admin Network Policy is available as a Technology Preview feature.
 
 You can enable `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic.
 
+[id="ocp-4-14-networking-http-headers"]
+==== Set or delete specific HTTP headers using an Ingress Controller or Route object
+
+Certain HTTP request and response headers can now be set or deleted either globally using an Ingress Controller or for specific routes. The headers you can set or delete include:
```
Comment (skrthomas) [inline release_notes/ocp-4-14-release-notes.adoc:332]:
```suggestion
Certain HTTP request and response headers can now be set or deleted either globally using an Ingress Controller or for specific routes. The headers you can set or delete include the following:
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -326,6 +326,22 @@ Admin Network Policy is available as a Technology Preview feature.
 
 You can enable `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources, which are part of the Network Policy V2 API, in clusters running the OVN-Kubernetes CNI plugin. Cluster administrators can apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Network administrators can secure clusters by enforcing network traffic controls that cannot be overridden by users. Network administrators can enforce optional baseline network traffic controls that can be overridden by users in the cluster, if necessary. Currently, these APIs support only expressing policies for intra-cluster traffic.
 
+[id="ocp-4-14-networking-http-headers"]
+==== Set or delete specific HTTP headers using an Ingress Controller or Route object
```
Comment (skrthomas) [inline release_notes/ocp-4-14-release-notes.adoc:330]:
```suggestion
==== Set or delete specific HTTP headers using an Ingress Controller or a Route object
```

**Review by skrthomas — COMMENTED** [review]
Just a couple nits, LGTM otherwise. Nice work :)

**Pull request merged**